fixture_magic/utils.py

```diff
@@ -15,6 +15,8 @@ def reorder_json(data, models, ordering_cond={}):
     :param ordering_cond: a key to sort within a model
     :return: the ordered JSON
     """
+    if ordering_cond is None:
+        ordering_cond = {}
```

Comment:
Agreed... Apparently without it, the argument can produce issues tho if changed:
```This inspection detects when a mutable value as list or dictionary is detected in a default value for an argument. Default argument values are evaluated only once at function definition time, which means that modifying the default value of the argument will affect all subsequent calls of the function.```